### PR TITLE
fix: broken build in todo app

### PR DESCRIPTION
**Pull request by m4thieulavoie**
# Pull Request

Fixes https://github.com/microsoft/fast/issues/4939

## 📖 Description

This PR fixes the broken start command in the todo example app of the monorepo

### General

- [x] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

File: examples/todo-app-tutorial/src/design-property-panel.ts [hunk 1]
```diff
@@ -144,15 +149,17 @@ const styles = css`
 })
 export class DesignPropertyPanel extends FASTElement {
     @observable provider!: any;
+
+    neutralLayerL1Behavior: typeof neutralLayer1 = neutralLayer1;
+
     providerChanged() {
-        this.provider.registerCSSCustomProperty(neutralLayerL1Behavior);
+        this.provider.registerCSSCustomProperty(this.neutralLayerL1Behavior);
```
Comment (m4thieulavoie) [inline examples/todo-app-tutorial/src/design-property-panel.ts:156]: I need help on this line. `registerCSSCustomProperty` has been deprecated, and I'm not sure which is the best approach to replace this bit of code. All the rest seems to work fine

Comment (nicholasrice) [reply examples/todo-app-tutorial/src/design-property-panel.ts:156]: That custom property should be added by the DesignToken automatically so can you just remove this line?

Comment (m4thieulavoie) [reply examples/todo-app-tutorial/src/design-property-panel.ts:156]: @nicholasrice does that mean I can remove every reference of `neutralLayerL1Behavior` ? Or simply this line?

File: examples/todo-app-tutorial/src/main.ts [hunk 2]
```diff
@@ -1,23 +1,28 @@
...
 import { DesignPropertyPanel } from "./design-property-panel";
 
-FluentDesignSystemProvider;
-FluentButton;
-FluentCheckbox;
-FluentTextField;
-FluentCard;
-FluentSlider;
-FluentSliderLabel;
+DesignSystem.getOrCreate()
```
Comment (EisenbergEffect) [inline examples/todo-app-tutorial/src/main.ts:15]: Let's update to use the new API `provideFluentDesignSystem()` and remove the `withPrefix(...)` calle.

File: examples/todo-app-tutorial/src/design-property-panel.ts (hunk 1 above)
Comment (nicholasrice) [reply examples/todo-app-tutorial/src/design-property-panel.ts:156]: You should be able to remove every line. You may need to assign `fillColor` to `neutralLayerl1` instead, but that would only happen once.

Comment (m4thieulavoie) [reply examples/todo-app-tutorial/src/design-property-panel.ts]: Should be fixed!

Comment (nicholasrice) [reply examples/todo-app-tutorial/src/design-property-panel.ts]: thanks!

**Comment by nicholasrice** [comment]
@m4thieulavoie Apologies for the delay, I'll find time to pull this down and finish the review today.

**Comment by nicholasrice** [comment]
@m4thieulavoie I pulled this down to have a look and after poking around I think there are some larger changes that need to happen to get this example in shape. If you don't mind, I'll push some commits over the top of this and highlight what I did and why. I may not get to it today but will try to get them up by the end of the week.

**Review by nicholasrice — CHANGES_REQUESTED** [review]
Per above I want to make some changes before this goes in. No action needed from you at this point but I'll push some changes over the top and we should be able to get this in soon. Thanks!

File: examples/todo-app-tutorial/src/design-property-panel.ts [hunk 3]
```diff
@@ -136,28 +141,76 @@ const styles = css`
...
-        this.provider.registerCSSCustomProperty(neutralLayerL1Behavior);
-        this.provider.style.setProperty(
-            "background-color",
-            `var(--${neutralLayerL1Behavior.name})`
-        );
-        this.provider.backgroundColor = (neutralLayerL1Behavior.value as any)(
-            this.provider.designSystem
+    @observable app!: HTMLElement;
+
+    // @designToken(baseLayerLuminance)
```
Comment (nicholasrice) [inline examples/todo-app-tutorial/src/design-property-panel.ts:168]:
```suggestion
```

**Review by nicholasrice — APPROVED** [review]
Okay I made the updates. The app wasn't leveraging design tokens and some of the values weren't getting parsed correctly.

I updated the control panel controls to set observable properties on the element, which notifies and updates design tokens for the `<todo-app>` element. 

Thanks for your work on this, I think it is good-to-go.

**Review by chrisdholt — APPROVED** [review]

**Pull request merged**